### DISCUSSION TITLE
How to update client if you have missed the update dialog?

### DISCUSSION THREAD

**Discussion by njmulsqb** [tutao/tutanota#4112] [Q&A] [answered]
Hi,

Generally when you open tutanota desktop client and if the update is available, it shows you a dialog stating something like "click here to update" but is there anyway to update the client if you miss that dialog for some reason?

**Comment by johnbotris** [comment, marked as answer]
if you go into "Settings > Desktop" there is an entry there where you can install the update

**Reply by charlag** [reply]
Or just restart the client
